compiler/damlc/daml-ide-core/src/Development/IDE/Core/Rules/Daml.hs

```diff
@@ -428,6 +456,7 @@ ofInterestRule opts = do
         -- updated.
         let scenarioFiles = files `Set.union` vrFiles
         gc scenarioFiles
+        let openVRsForFile file = Set.filter (\vr -> vrScenarioFile vr == file) openVRs
```

Comment:
Probably not a huge deal but turning this into a `Map NormalizedFilePath [VirtualResource]` would be both faster and imho clearer, e.g., `openVRsByFile = Map.fromList (map (\vr -> (vrScenarioFile vr, vr)) openVRs)`.

Reply:
Thanks!